server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -566,6 +598,50 @@ fn create_filtered_outgoing_sync_query(
     query
 }
 
+// This is a manual sync to fetch all records for a specific patient
+// Managed via own cursor
+fn create_filtered_outgoing_patient_sync_query(
+    earliest: u64,
+    sync_site_id: i32,
+    fetch_patient_id: String,
+) -> BoxedChangelogQuery {
+    let mut query = changelog_deduped::table
+        .left_join(name_link::table)
+        .filter(changelog_deduped::cursor.ge(earliest.try_into().unwrap_or(0)))
+        .into_boxed();
+
+    let active_stores_for_site = store::table
```

Comment:
I actually agree here with the dry choice, although if active_store_for_site was a new method it would have been good too, but something about this simple active_store_for_site variable makes more sense (even though it appears in two places)